src/main/java/com/spartronics4915/frc2020/Robot.java

```diff
@@ -101,7 +102,7 @@ public void disabledInit()
     {
         // TODO: verify call to DISABLED bling state
         Logger.notice("@disabledInit: Requested BlingState.BLING_COMMAND_DISABLED");
-        // mRobotContainer.setBlingState(BlingState.BLING_COMMAND_DISABLED);
+        LED.getInstance().setBlingState(BlingState.BLING_COMMAND_DISABLED);
```

Comment:
@randomgrace @MayaShaw I also suggest adding 
* `BLING_COMMAND_STARTUP` to `teleopInit()` 
* `BLING_COMMAND_AUTOMODE` to `autonomousInit()`

@j-james LED's `setBlingState()` is not a blocking function. Recommend using this method (`LED.getInstance().setBlingState()`) where we need to change LED state in the code vs. scheduling a command. 

@j-james please merge this code.